rudder-webapp/SOURCES/Makefile

```diff
@@ -57,11 +57,15 @@ localdepends: ./rudder-sources ./maven/bin/mvn ./rudder-users.xml ./rudder-doc .
 ./rudder-users.xml: ./rudder-sources
 	cp ./rudder-sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml ./rudder-users.xml
 
+./.gitignore: ./configuration-repository-gitignore
+	cp ./configuration-repository-gitignore ./.gitignore
```

Comment:
Why do you copy the file here instead of just calling it with its original name in the package definition ?